core/src/main/java/com/exadel/aem/toolkit/core/lists/utils/ListHelper.java

```diff
@@ -90,6 +105,62 @@ private ListHelper() {
             .collect(Collectors.toList());
     }
 
+    public static Page createList(ResourceResolver resourceResolver, String parentPath, String pageName, List<SimpleListItem> values)
+        throws WCMException, PersistenceException {
+        List<ValueMap> listItemsProperties = values.stream()
+            .map(item -> createListItemProperties(item.getTitle(), item.getValue()))
+            .collect(Collectors.toList());
+
+        return create(resourceResolver, parentPath, pageName, listItemsProperties);
+    }
+
+    public static Page createResourceList(ResourceResolver resourceResolver, String parentPath, String pageName, List<Resource> values)
```

Comment:
I suggest naming all variations of this method `createList` , as they produce the same result, but just take different arguments.